src/SIS_dyn_trans.F90

```diff
@@ -453,6 +454,13 @@ subroutine SIS_dynamics_trans(IST, OSS, FIA, IOF, dt_slow, CS, icebergs_CS, G, U
           call set_wind_stresses_C(FIA, ice_cover, ice_free, WindStr_x_Cu, WindStr_y_Cv, &
                                    WindStr_x_ocn_Cu, WindStr_y_ocn_Cv, G, US, CS%complete_ice_cover)
 
+          if (CS%lemieux_landfast) then
```

Comment:
I don't understand the point of putting in the option to use a older, deprecated style of the code with a new option.  I think that that the Warsaw_sum_order branch should be eliminated here.